cloudify_awssdk/efs/resources/mount_target.py

```diff
@@ -31,7 +31,7 @@
 SUBNET_ID = 'SubnetId'
 SUBNET_TYPE = 'cloudify.nodes.aws.ec2.Subnet'
 SUBNET_TYPE_DEPRECATED = 'cloudify.aws.nodes.Subnet'
-SECGROUP_TYPE = 'cloudify.aws.nodes.SecurityGroup'
+SECGROUP_TYPE = 'cloudify.nodes.aws.ec2.SecurityGroup'
```

Comment:
Do we need to support the old case for backward comp?

Reply:
I will update that